src/components/MaskedText.tsx

```diff
@@ -5,12 +5,16 @@ import { mask } from "../utils/mask";
 interface MaskedTextProps {
   children: string;
   mask: string;
+  type?: "custom" | "currency";
+  options?: any;
```

Comment:
Try to remove "any" from options type, and type it inplace instead

Reply:
Did it